pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -713,17 +713,6 @@ def build_model(self):
         self.set_degradation_variables()
         self.set_summary_variables()
 
-        # Massive hack for consistent delta_phi = phi_s - phi_e with SPMe
```

Comment:
bye bye!

Reply:
Where was the issue?

Reply:
I just made a new submodel for this. So now we have inverse butler-volmer for delta_phi_av and then another submodel for delta_phi. It's still a hack but now a "feature" instead of a "bug"

Reply:
I am also thinking of combining the interface and conductivity submodels into one. It's the most confusing part of the submodels at the moment, because the way the interface and conductivity submodels interact is different based on whether you do forward or inverse kinetics. Combining them would mean more duplicate code but I think clearer